protractor/e2e/support/second.po.ts

```diff
@@ -5,27 +5,31 @@ export class SecondPo extends BaseComponent{
     pageUrl = '/2';
     downloadChapterButton: ElementFinder = $('.hero-cta');
     headerLinks: ElementArrayFinder = $$('.nav-links a')
-
     stepsBlock: ElementArrayFinder= $$('div.step');
-
     firstnameInput: ElementFinder = $('.greensboro-field-name');
     emailInput: ElementFinder = $('.greensboro-field-email');
     errorMassege: ElementArrayFinder =  $$('.greensboro-error');
     submitButton: ElementFinder = $('.greensboro-field-submit');
     popupCloseButton: ElementFinder = $('.greensboro-CloseButton');
-    stepsBlock: ElementArrayFinder = $$('div.step');
-
-
     secondPageLink: ElementFinder = $('.navbar-brand');
     bookSection: ElementFinder = $('.curriculum');
     bookContentSection: ElementArrayFinder = $$('.curriculum-list-item');
-
-
     readTableContentButton: ElementFinder = $('.curriculum-btn');
     pdfContent: ElementFinder = $('[src*="media/ng2/ng-book-2-table-of-contents.pdf"]');
     infoCardSection: ElementFinder = $('.info-card.too-good');
     infoEmailInput: ElementFinder = $('.infusion-field-input-container');
     infoSubmitButton: ElementFinder = $ ('.input-group-btn [type="submit"]');
+//home work
+    menuItemBlog: ElementFinder= $ ('.nav-links a[href^="http"]');
```

Comment:
'a[href*="blog"]' - will be more stable because http attribute value could be not only in 1 element